config.json

```diff
@@ -658,7 +658,7 @@
         "name": "Scrabble Score",
         "uuid": "11771d47-1109-4579-a62b-e0b8e9583485",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["strings", "array-transformations"],
```

Comment:
Here is my solution : https://exercism.org/tracks/javascript/exercises/scrabble-score/solutions/TomPradat

I only put 2 prerequisites because `array-transformations` already "includes" many other concetps